File: problems/002-two-sum/swift/practice.swift
import Foundation

func practice(_ c: [String: Any]) -> Any {
    let nums = (c["nums"] as! [Any]).map { ($0 as! NSNumber).intValue }
    let target = (c["target"] as! NSNumber).intValue
    // Return the indices [i, j] (i < j) of the two numbers in nums that add up to target.
    return [Int]()
}

File: problems/002-two-sum/csharp/Practice.cs
using System.Collections.Generic;
using System.Text.Json;
class Practice : ISolution {
    public object Solve(Dictionary<string, object> c) {
        var target = ((JsonElement)c["target"]).GetInt64();
        var nums = (JsonElement)c["nums"];
        // Return the indices [i, j] (i < j) of the two numbers in nums that add up to target.
        return new int[0];
    }
}
